docs/getting-started/coverage.md

```diff
@@ -0,0 +1,54 @@
+# Scanning Coverage
+
+## Programming languages
```

Comment:
What if categorizing by scanners? Or is the purpose of this page to show the big picture without the details?

```suggestion
## Vulnerabilities
### OS packages
### Language-specific packages

## Misconfiguration

## License
```

Reply:
yes, we should highlight that there are 4 different scanner in Trivy since we refer to it on other docs pages